### ISSUE TITLE
Checkbox and Radio should be accessible

### ISSUE THREAD

**Issue by interactivellama** [ExactTarget/fuelux#1972]
Checkbox and Radio Button templates should include `tabindex="0"` so that in applications that use CSS to hide the input element (eg Lightning Theme) they are still tab-able.

**Comment by cormacmccarthy** [comment]
@interactivellama I'm having trouble replicating this issue. You can see it working just fine here:

http://fuelux-edge.herokuapp.com/

**Comment by interactivellama** [comment]
It's not working here:
http://fuelux.herokuapp.com/controls/checkboxes

I'm thinking it may be Bootstrap or MCtheme/Fusion or Lightning (which isn't in use in Email).

I will see if I can track down the library that is causing it. For all I know it's Fuel UX 2 or something old like that.

**Comment by cormacmccarthy** [comment]
It is re-create-able in Lightning Theme, because LT sets `visibility: hidden;` on the input.

**Comment by interactivellama** [comment]
This bug appears to only be a LT and MCT issues. Feel free to close if you want.

**Comment by cormacmccarthy** [comment]
Well, I'm adding a check to error to the console if someone does those things (see #1996 ) and then I'm going to go in and fix LT and MCT.

**Comment by interactivellama** [comment]
So we are on board with a fix in MCT. I thought that wasn't being touched anymore.

**Comment by cormacmccarthy** [comment]
It is officially _not_ supported. That doesn't stop devs from issuing PRs on it though. I haven't looked at it yet, but if it is as simple as just trashing the `visibility: hidden;` declaration, I'm going to do that.
